book/src/user/lightwalletd.md

```diff
@@ -0,0 +1,173 @@
+
+Tweak the following option in order to prepare for lightwalletd setup.
+
+### RPC section
+[#rpc-section]: #rpc-section
+
+This change is required for zebra to behave as an RPC endpoint. The standard port for RPC endpoint is `8232`.
+
+```
+[rpc]
+listen_addr = "127.0.0.1:8232"
+```
+
+## Sync Zebra
+[#sync-zebra]: #sync-zebra
+
+With the configuration in place you can start synchronizing Zebra with the Zcash blockchain. This may take a while depending on your hardware.
+
+```console
+zebrad start
+```
+
+Zebra will display information about sync process:
+
+```console
+...
+zebrad::commands::start: estimated progress to chain tip sync_percent=10.783 % 
+...
+```
+
+Until eventually it will get there:
+
+```console
+...
+zebrad::commands::start: finished initial sync to chain tip, using gossiped blocks sync_percent=100.000 % 
+...
+```
+
+You can interrupt the process at any time with `ctrl-c` and Zebra will resume the next time at around the block you were downloading when stopping the process.
+
+When deploying for production infrastructure, the above command can/should be implemented as a server service or similar configuration. 
+
+For implementing zebra as a service please see [here](https://github.com/ZcashFoundation/zebra/blob/main/zebrad/systemd/zebrad.service).
+
+## Download and build lightwalletd
+[#download-and-build-lightwalletd]: #download-and-build-lightwalletd
+
+While you synchronize Zebra you can install lightwalletd. There are several versions of lightwalletd in different repositories:
+
+- [The original version](https://github.com/zcash/lightwalletd)
+- [The version used by zecwallet-lite and zecwallet-cli](https://github.com/adityapk00/lightwalletd)
+
+Most of the testing related to Zebra and lightwalletd was done with the zecwallet version in the [adityapk00](https://github.com/adityapk00) repository.
+
+Before installing lightwalletd, you need to have `go` in place. Please visit the [go install page](https://go.dev/doc/install) with download and installation instructions.
+
+With go installed and in your path, download and install lightwalletd:
+
+```console
+git clone https://github.com/adityapk00/lightwalletd
+cd lightwalletd
+make
+make install
+```
+
+If everything went good you should have a `lightwalletd` binary in `~/go/bin/`.
+
+## Sync lightwalled
+[#sync-lightwalletd]: (#sync-lightwalletd)
+
+Please make sure you have zebrad running (with RPC endpoint and up to date blockchain) to synchronize lightwalletd.
+
+- `lightwalletd` requires a `zcash.conf` file located somewhere, however this file can be empty if you are using the default zebra rpc endpoint (`127.0.0.1:8232`).
+
+- For production setups `lightwalletd` requires a `cert.pem`. For more information on how to do this please [see here](https://github.com/zcash/lightwalletd#production-usage).
+
+- `lightwalletd` can run without the certificate (with the `-no-tls-very-insecure` flag) however this is not recommended for production environments.
```

Comment:
```suggestion
- `lightwalletd` can run without the certificate (with the `--no-tls-very-insecure` flag) however this is not recommended for production environments.
```

Reply:
Hi @AArnott , thank you for the suggestions. As this pull request was already merged we need to open your suggestions in a new one. Do you want to make the suggested changes at https://github.com/ZcashFoundation/zebra/blob/main/book/src/user/lightwalletd.md and open a pull request wioth them or prefer me to do it ?

Reply:
If you don't mind migrating these to a new PR, I'd appreciate it. I'll send further feedback in a fresh one.

Reply:
Created https://github.com/ZcashFoundation/zebra/pull/4566